content/docs/command-reference/check-ignore.md

```diff
@@ -37,6 +37,12 @@ ones that are ignored indeed are printed back.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+- `--stdin` - read paths interactively from standard input instead of providing
+  `targets`. Exit with `Ctrl+C`.
```

Comment:
```suggestion
- `--stdin` - read paths from standard input instead of providing `targets`.
  Useful for interactive debugging (exit with `Ctrl+C`) and POSIX pipes.
```